### DISCUSSION TITLE
Roadmap / Timelines - Large migration from V3 to V4, or wait till V5?

### DISCUSSION THREAD

**Discussion by leobudima** [ant-design/ant-design#32854] [General]
Hi and thanks for such an immense project that has really gotten my team a lot of value! 

We have a fairly large project, in which we are still stuck on V3 due to various functional / styling overrides we have that are causing issues when upgrading to V4. We want to plan work to migrate to V4, but I'm curious to know if there are any existing plans for V5, any timelines or similar, that would help decide whether to spend the work now, and then do it again for V5, or if V5 is planned in the next year or so, wait till then and do it only once to migrate to V5.

So, in short - is there any information for when V5 is planned?

Thank you for any information!
